src/data/finset/basic.lean

```diff
@@ -2238,6 +2238,42 @@ finset.strong_induction_on s $ λ s,
 finset.induction_on s (λ _, h₀) $ λ a s n _ ih, h₁ a s n $
 λ t ss, ih _ (lt_of_le_of_lt ss (ssubset_insert n) : t < _)
 
+/-- Suppose that, given `s.card ≤ n` and objects defined on all strict supersets of card less than
+`n` of any finset, one knows how to define an object on `s`. Then one can inductively define an
+object on all finsets of card less than `n`, starting from finsets of card `n` and iterating. This
+can be used either to define data, or to prove properties. -/
```

Comment:
The wording of this feels really awkward to me – it took me a couple reads to get it. Maybe something like "given that `p s` can be defined `p` of all of the supersets of `s` with cardinality less than `n`, you can construct `p s` for all finsets less than `n`.

Reply:
Yeah, it's kind of hard to convey the meaning without mimicking the quantifiers. Is that better now?

Reply:
Yep, seems better now!